src/tool.rs

```diff
@@ -62,6 +62,11 @@ impl Tool {
             Ok(p) => p,
         };
 
+        if !path.exists() {
+            eprintln!("Failed to find tool: {}\nat: {}\nConsider `rustup component add llvm-tools-preview`", self.name(), path.to_string_lossy());
```

Comment:
```suggestion
            eprintln!("Could not find tool: {}\nat: {}\nConsider `rustup component add llvm-tools-preview`", self.name(), path.to_string_lossy());
```

I'd like it if this was a hint/suggestion ala `eyre::Report` or whatever, but no real need for it and it's another dependency to add.